README.md

```diff
@@ -89,6 +68,13 @@ rails generate yoti:install
 
 The generated initialisation file can be found in `config/initializers/yoti.rb`.
 
+## Setup
+
+For each service you will need:
+
+* Your Client SDK ID, generated by [Yoti Hub](https://hub.yoti.com) when you create (and then publish) your app. 
+* Your .pem file. This is your own unique private key which your browser generates from the [Yoti Hub](https://hub.yoti.com) when you create an application.
+
```

Comment:
Do you think we could combine "Setup" and "Configuration"? They look very similar. 

Or at least we could just reference the Setup section instead of detailing what the `YOTI_CLIENT_SDK_ID` and `YOTI_KEY_FILE_PATH` are?

Reply:
Removed duplicated descriptions in f881886